.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "root": true,
+  "extends": ["cesium/browser"],
+  "plugins": ["html", "es"],
+  "rules": {
+    "no-unused-vars": ["error", { "vars": "all", "args": "none" }]
+  },
+  "overrides": [
```

Comment:
Maybe this is picky, but would it make sense to default to `cesium/node` by default, then override the config with `cesium/browser` for the `src` folder? Since typically files at the root are node scripts, and `src` files are intended to be run in the app in the browser.

Reply:
@ggetz I just split the files instead